### DISCUSSION TITLE
Increase track width

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#654] [Q&A] [answered]
If one follows a track, it's helpful to see the track and the underlaying map. This results in a transparent track. The track width is currently limited to 200%. Increase this limit to 500%.

Half-transparent track (width 200%):
![Screenshot_20241026_092025_Cruiser](https://github.com/user-attachments/assets/6e84fa1c-94c1-472a-8e56-75fef2ca9bdc)

Current track width limit:
![Screenshot_20241026_092439_Cruiser](https://github.com/user-attachments/assets/22e5d3a4-ddc9-447b-81a6-bdde598b87e1)

**Comment by devemux86** [comment]
Thanks for the suggestion.

The track in the screenshot cannot be seen. Probably a huge width will not help if too much transparency is set.

How the other apps handle this? Can you post some screenshots with other apps and the suggested large width?

**Comment by printmaps** [comment]
Cruiser:
![Screenshot_20241026_101855_Cruiser](https://github.com/user-attachments/assets/1d54ac0b-bff2-470b-9efe-c284be0c2a88)

Locus Map:
![Screenshot_20241026_101735_Locus Map Classic](https://github.com/user-attachments/assets/c77a417e-b5ee-4a8c-8b64-674efccec37d)

**Comment by printmaps** [comment]
Just for clarification: Meant is the width and transparency of an already recorded track which it put as overlay onto the map.

If I follow a track I want to see three thing:
- all details of the map
- the track I want to follow (as transparent and width line)
- the currently recorded track (as solid line)

**Reply by devemux86** [reply]
Track width exists in the settings at 2 areas:
- "Routing | Appearance" for the overlay tracks
- "GPS recording | Appearance" for the recorded track

**Comment by devemux86** [comment]
I can increase the max percentage in the track width setting.

Note that the default track width was half the route width from the beginning,
so the track can be painted over the route and we could still see the route line.

We could also increase the default track width to be the same as the route width?

**Comment by devemux86** [comment]
Max track width 500% is available for testing in **Cruiser 5.0.1-b12**.

**Comment by devemux86** [comment]
Default track width is now 3/4 of the default route width, available in Cruiser 5.0.1-b13.

**Comment by devemux86** [comment, marked as answer]
- Implemented in https://github.com/devemux86/cruiser/discussions/641
